Ascend.java

```diff
@@ -2,6 +2,26 @@
 // and prints them in increasing order.
 public class Ascend {
 	public static void main(String[] args) {
-		// Replace this comment with your code
+		// get input number for a limit value and random 3 numbers between 0 and the input
+		int number=Integer.parseInt(args[0]);
+		number=number+1;
+		double num1=Math.random()*number;
+		double num2=Math.random()*number;
+		double num3=Math.random()*number;
+		//save the values in another variables
+		int num4=(int)num1;

```

Comment:
no need to declare another 3 variables on top of the first three, 
can use:
int num1=(int)Math.random()*number;